site/content/en/docs/Installation/_index.md

```diff
@@ -96,46 +96,89 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 A [cluster][cluster] consists of at least one *cluster master* machine and multiple worker machines called *nodes*: [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
 
+{{% feature expiryVersion="1.1.0" %}}
 ```bash
 gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.12 \
   --tags=game-server \
   --scopes=gke-default \
   --num-nodes=4 \
   --machine-type=n1-standard-4
 ```
+{{% /feature %}}
+{{% feature publishVersion="1.1.0" %}}
+```bash
+gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.12 \
+  --tags=game-server \
+  --scopes=gke-default \
+  --num-nodes=4 \
+  --no-enable-autoupgrade \
```

Comment:
Any reason we can't add this to the current documentation? There's no downside to setting `--no-enable-autoupgrade` at this stage, is there?

Reply:
No particular reason. Since it'll be published in less than a week anyway we can just push it live right away.